pyproject.toml

```
@@ -0,0 +1,34 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=77.0"]
+
+[project]
+name = "ha-ffmpeg"
+version = "3.2.2"
+license = "BSD-3-Clause"
+description = "A library that handling with ffmpeg for home-assistant"
```

Side note: We could also correct the grammar here.
```suggestion
description = "A library handling ffmpeg for home-assistant"
```